### ISSUE TITLE
Ender IO recipe file do not have solar panel

### ISSUE THREAD

**Issue by Joccob** [MachineMuse/MachineMusePowersuits#673]
As mentioned in the title, I find that the solar panel's recipe is missing when only enderio is installed.

**Comment by Xen1290** [comment]
```
    "ingredients" : [
        [ { "oredictName" : "dyeBlue" }, { "oredictName" : "blockGlass" }, { "oredictName" : "dyeBlue" } ],
        [ { "oredictName" : "blockGlass" }, { "oredictName" : "blockLapis" }, { "oredictName" : "blockGlass" } ],
        [ { "oredictName" : "circuitBasic" }, { "oredictName" : "circuitAdvanced" }, { "oredictName" : "circuitBasic" } ]
    ],
    "result" : {
        "oredictName" : "componentSolarPanel"
```

Try with this recipe in a crafting table, maybe only nei doesn't show it, because in the EnderIO recipe file this recipe exist.

**Comment by Xen1290** [comment]
Edit: I found the issue. I'll make a Pull Request to fix this. Thanks for the report.

**Comment by flowtender** [comment]
This is fixed, right? #687

**Comment by Xen1290** [comment]
Yes @MachineMuse
